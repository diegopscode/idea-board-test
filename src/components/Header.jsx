import React from 'react'

const HeaderApp = () => (
	<header className="app-header container">
		<h1 className="title title-header">Idea Board</h1>
	</header>
)

export default HeaderApp
