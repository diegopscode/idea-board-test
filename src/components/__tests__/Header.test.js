import React from 'react'
import renderer from 'react-test-renderer'

import Header from '../Header'

describe("HeaderComponent", () => {
    test('snapshot renders header component', () => {
        const component = renderer.create(<Header />)
        let header = component.toJSON()
        expect(header).toMatchSnapshot()
    })
})