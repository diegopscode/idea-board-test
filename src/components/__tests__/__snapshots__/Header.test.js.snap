// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderComponent snapshot renders header component 1`] = `
<header
  className="app-header container"
>
  <h1
    className="title title-header"
  >
    Idea Board
  </h1>
</header>
`;
